core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -173,8 +173,6 @@ protected[controller] class RestAPIVersion(apipath: String, apiversion: String)(
                     webexp.routes()
                 } ~ {
                     swaggerRoutes
-                } ~ options {
```

Comment:
right - you can invert the order of route matching so web comes first (it's a stricter match).
alternatively make the option matching in the top route more strict (not `/web` prefix).